test/test_notebooks.py

```diff
@@ -20,41 +53,31 @@ def test_explain_binary_classification_local():
 
 @pytest.mark.notebooks
 def test_explain_regression_local():
-
     notebookname = "explain-regression-local"
-    input_notebook = "notebooks/" + notebookname + ".ipynb"
-    output_notebook = "./test/" + notebookname + ".output.ipynb"
-
-    pm.execute_notebook(input_notebook, output_notebook)
+    input_notebook = input_notebook_path(notebookname)
+    output_notebook = output_notebook_path(notebookname)
+    processed_notebook = processed_notebook_path(notebookname)
+    test_values = {"local_imp": "sorted_local_importance_names"}
```

Comment:
so it stores the value of `sorted_local_importance_names` in a dictionary under `local_imp`?

Reply:
local_imp is a scrap.  There is a cell added to the notebook that "glues" the sorted_local_importance_names to the local_imp scrap.  The scrap is then used in the tests to validate the data in sorted_local_importance_names.